wp-content/themes/mojintranet/inc/cache.php

```diff
@@ -4,8 +4,8 @@
   }
 
   function dw_clear_headers() {
-    $cache_timeout = 90;
-    
+    $cache_timeout = 60;
```

Comment:
not sure if it's helpful to make a small comment indicating that it's referring to seconds?

Reply:
Cache-Control header entries are always in seconds...